### PR TITLE
Editorial: change [[localeMatcher]] to [[LocaleMatcher]] to align w/ 262

### PR DESCRIPTION
**Pull request by ben-allen**
(very) partial fix [#81](https://github.com/tc39/ecma402/issues/81) . this one does not seem to be exposed to [[&lt;dynamic&gt;]] access problems. submitted to get reality check on my judgement here.

### CODE REVIEW

File: spec/datetimeformat.html [hunk 1]
```diff
@@ -203,7 +203,7 @@ <h1>Internal slots</h1>
           [[LocaleData]].[[&lt;_locale_&gt;]].[[hc]] must be &laquo; *null*, *"h11"*, *"h12"*, *"h23"*, *"h24"* &raquo;.
         </li>
         <li>
-          [[LocaleData]].[[&lt;_locale_&gt;]].[[hourCycle]] must be a String value equal to *"h11"*, *"h12"*, *"h23"*, or *"h24"*.
```
Comment (ryzokuken) [inline spec/datetimeformat.html:206]: This one seems to be imbalanced. I suppose there is an access to this `[[hourCycle]]` field that needs to be updated too?

Comment (ljharb) [reply spec/datetimeformat.html:206]: generally every commit landed to main needs to be correct, so it might need to all go in the same PR

**Comment by sffc** [comment]
@ben-allen This is editorial, correct? Please say so in the PR title

**Comment by ljharb** [comment]
@ben-allen why is this closed?

**Comment by ben-allen** [comment]
@ljharb Apologies -- replaced this one with #766.

**Pull request closed without merging**